scripts/use_model.py

```diff
@@ -161,14 +167,17 @@ def main():
     # 2. load data
     print("Loading data and creating DataLoaders")
 
-    dw = data_utils.DataHandler(data_source, model_args.model_args)
+    dw = data_utils.DataHandler(data_source, model_args.model_args, cache_class)
```

Comment:
'cache_class' argument is no longer supported

Reply:
Fixed